stable_baselines/common/policies.py

```diff
@@ -256,13 +266,8 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=Fals
                 pi_latent = extracted_features
                 vf_latent = extracted_features
             else:
-                activ = tf.tanh
-                processed_x = tf.layers.flatten(self.processed_x)
-                pi_h = processed_x
-                vf_h = processed_x
-                for i, layer_size in enumerate(layers):
-                    pi_h = activ(linear(pi_h, 'pi_fc' + str(i), n_hidden=layer_size, init_scale=np.sqrt(2)))
-                    vf_h = activ(linear(vf_h, 'vf_fc' + str(i), n_hidden=layer_size, init_scale=np.sqrt(2)))
+                vf_h = mlp(self.processed_x, layers=layers, layer_name_preffix="vf_fc", **kwargs)
```

Comment:
I would rather pass the kwargs explicitely

Reply:
Fair enough... I still don't really like that thing but I don't see better solution.